examples/simple/lint/L-CEIE1-bad.c

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) 2015 Runtime Verification, Inc. (RV-Match team). All Rights Reserved.
+
+#include <stdlib.h>
+#include <stdio.h>
+
+int main()
```

Comment:
You had two lines before the main function until now, and now you have one line.

@msaxena2 , can you please go through my previous comments and address them one by one, telling whether you took care of it and how, or whether you have not and why?  Otherwise I'm wasting a lot of time re-checking everything that I've already reported.